bundles/org.openhab.binding.miio/src/main/java/org/openhab/binding/miio/internal/basic/Conversions.java

```diff
@@ -121,9 +123,28 @@ public static JsonElement tankLevel(JsonElement value12) throws ClassCastExcepti
         }
     }
 
+    public static JsonElement getJsonElement(String element, JsonElement responseValue)
+            throws ClassCastException, IllegalStateException {
```

Comment:
You'd better know before accessing it if it's the right type in the right state. Catching these is bad.

Reply:
maybe I'm overly causious here.. as the responsevalue is something that comes from the device and can have rather unexpected values I normally rather have a friendly message than an ugly stacktrace.

That said, I think the code indeed has little chance to generate these errors even in case of strange inputs so i'll remove it